### PR TITLE
refactor: ovms model download scripts refactor and cleanup

### PR DESCRIPTION
**Pull request by jim-wang-intel**
clean up the old model downloader and refactor the ovms model downloader to a better function structure in script

CLOSES: #409

## PR Checklist

- [x] Added label to the Pull Request for easier discoverability and search
- [x] Commit Message meets guidelines as indicated in the URL https://github.com/intel-retail/automated-self-checkout/blob/main/CONTRIBUTING.md
- [x] Every commit is a single defect fix and does not mix feature addition or changes
- [ ] Unit Tests have been added for new changes
- [ ] Updated Documentation as relevant to the changes
- [ ] All commented code has been removed
- [ ] If you've added a dependency, you've ensured license is compatible with repository license and clearly outlined the added dependency.
- [ ] PR change contains code related to security
- [ ] PR introduces changes that breaks compatibility with other modules (If YES, please provide details below)

## Issue this PR will close

close: #**409**

## Test Instructions if applicable

- git clone this PR
- for unit-test: 
   ```console
        cd ./download_models; ./testDownloadOVMSModels.sh
  ```
should see ALL PASSED
- for integration tests: run smoke tests `make run-smoke-tests`  and should see ALL PASSED

### CODE REVIEW

**Review by brian-intel — APPROVED** [review]

File: download_models/downloadOVMSModels.sh [hunk 1]
```diff
@@ -1,18 +1,12 @@
 #!/bin/bash
 #
-# Copyright (C) 2023 Intel Corporation.
+# Copyright (C) 2024 Intel Corporation.
```
Comment (NeethuES-intel) [inline download_models/downloadOVMSModels.sh:3]: this may be a change that will have to be done in other files as well. Out of scope for this PR though.

**Review by NeethuES-intel — APPROVED** [review]
LGTM

File: download_models/downloadOVMSModels.sh (hunk 1 above)
Comment (jim-wang-intel) [reply download_models/downloadOVMSModels.sh:3]: I just changed whenever I touched the files

**Review by antoniomtz — APPROVED** [review]
LGTM. passes tests

**Pull request merged**